app/app-config.yaml

```diff
@@ -179,10 +179,10 @@ catalog:
   # Locations configured to import static data into the catalog
   locations:
     - type: url
-      target: https://github.com/DEFRA/adp-software-templates/blob/main/catalog-model/all.yaml
+      target: https://github.com/DEFRA/adp-software-templates/blob/1.0.0/catalog-model/all.yaml
```

Comment:
I think we should read the version from config.

Reply:
yes app config I think?

Reply:
@ppotturi @DaRoszko Done